documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc

```diff
@@ -23,8 +23,10 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
 do
   sudo virsh destroy $i;
   sudo virsh undefine $i;
-  sudo virsh vol-delete $i --pool default;
+  sudo virsh vol-delete $i --pool $i;
   sudo virsh vol-delete $i.ign --pool default;
+  sudo virsh pool-destroy $i;
```

Comment:
ok from me

Reply:
It won't be able to destroy a pool of $i as its not the actual name. Going back to your above concern.

Reply:
I meant to destroy it, but yes, using 'default'